hack/multi-arch.sh

```diff
@@ -18,15 +18,46 @@
 #
 
 COMMAND=$1
+# We are formatting the architecture name here to ensure that
+# it is consistent with the platform name specified in ../.bazelrc
+function format_archname() {
+    local local_platform=$(uname -m)
+    local platform=$1
+
+    if [ $# -lt 1 ]; then
+        echo ${local_platform}
+    else
+        case ${platform} in
+        x86_64* | i?86_64* | amd64*)
```

Comment:
Are the `*`s needed here? And what is `i?86_64*` supposed to match?

Reply:
Yes, you are right.